modules/ROOT/pages/_partials/block-caveats.adoc

```diff
@@ -110,4 +110,16 @@ Refer to xref:server:developer-preview:preview-mode.adoc[Developer Preview Mode]
 Use only *Couchbase* bucket types in _Couchbase for Mobile and Edge_.
 We do not support the use of Couchbase Server's *Ephemeral* or *Memcached* bucket types -- for more on bucket types see:  Couchbase Server xref:server:learn:buckets-memory-and-storage/buckets.adoc[bucket types].
 
-// end::ephemeral-buckets[]
\ No newline at end of file
+// end::ephemeral-buckets[]
+
+// tag::cbs6.0ke-xattrs[]
+[NOTE]
+--
+Users of Couchbase Server 6.0 should ensure they have addressed the known error (https://issues.couchbase.com/browse/MB-41255[MB-41255]) by upgrading to one of the recommended Couchbase Server versions (6.0.5, 6.5.2, or 6.6.1).
```

Comment:
I think "known issue" is a better term than "known error" for this line and the line below.

Reply:
Resolved --https://github.com/couchbase/docs-sync-gateway/pull/391/commits/3f0669141b06b1b0bf162027be3d2033beda328b